Missionframework/modules/00_init/fnc/fn_init_loadPresets.sqf

```diff
@@ -26,7 +26,7 @@
 */
 
 // Load Blufor preset
-switch (KPLIB_preset_blufor) do {
+switch (KPLIB_param_presetPlayer) do {
```

Comment:
Maybe this param could be named `KPLIB_param_presetFriendly` as it effects the whole faction with all AI's and not only the players.
Could be missunderstood.